docs/source/build-run-openvino.md

```diff
@@ -0,0 +1,169 @@
+# Building and Running ExecuTorch with OpenVINO Backend
+
+In this tutorial we will walk you through the process of setting up the prerequisites, building OpenVINO backend library, exporting `.pte` models with OpenVINO optimizations, and executing the exported models on Intel hardware.
+
+<!----This will show a grid card on the page----->
+::::{grid} 2
+:::{grid-item-card}  What you will learn in this tutorial:
+:class-card: card-prerequisites
+* In this tutorial you will learn how to lower and deploy a model with OpenVINO.
+:::
+:::{grid-item-card}  Tutorials we recommend you complete before this:
+:class-card: card-prerequisites
+* [Introduction to ExecuTorch](intro-how-it-works.md)
+* [Setting up ExecuTorch](getting-started-setup.md)
+* [Building ExecuTorch with CMake](runtime-build-and-cross-compilation.md)
+:::
+::::
+
+## Introduction to OpenVINO
+
+[OpenVINO](https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/overview.html) is an open-source toolkit designed to enhance AI inference on Intel hardware by reducing latency and increasing throughput while preserving accuracy. It optimizes hardware utilization and simplifies AI development and deep learning integration across domains such as computer vision, large language models (LLMs), and generative AI.
+
+OpenVINO is integrated as an Executorch delegate to accelerate AI applications deployed with Executorch APIs. 
+
+## Supported Hardware
+
+OpenVINO backend supports the following hardware:
+
+- Intel CPUs
+- Intel integrated GPUs
+- Intel discrete GPUs
+- Intel NPUs
+
+## Instructions for Building OpenVINO Backend
+
+### Prerequisites
+
+Before you begin, ensure you have openvino installed and configured on your system:
+
+#### TODO: Add instructions for support with OpenVINO release package
```

Comment:
If you gonna live this TODO as is then maybe call it out as "something not supported".

Reply:
Removed and added a note instead.